src/main/webapp/partials/instructorHelpGettingStarted.jsp

```diff
@@ -319,8 +333,16 @@
             <li>"session using template: team peer evaluation"</li>
             <div style="margin: 0 auto; padding: 0 50px;">
               <ul>
-                <li>Provides 5 standard questions for team peer evaluations</li>
-                <li> Allows you to modify/remove the given questions and add your own questions as required</li>
+                <li>Provides 5 questions for team peer evaluations, with the first question being a contribution question</li>
+                <li>Allows you to modify/remove the given questions and add your own questions as required</li>
+              </ul>
+            </div>
+            <li>session using template: team peer evaluation (optimized)</li>
+            <div style="margin: 0 auto; padding: 0 50px;">
+              <ul>
+                <li>Provides 5 standard questions for team peer evaluations, with the first question being a distribute points question</li>
+                <li>Allows you to modify/remove the given questions and add your own questions as required</li>
+                <li>Recommended template for new users (<a href="#fbConstSumOptions">distribute points questions</a> are generally easier to use than <a href="#fbContrib">contribution questions</a>)</li>
```

Comment:
This is not correct. The _optimized_ one is the one that contains the contribution question. You can also mention that contribution questions are optimized to obtain a more accurate view of the contribution distribution (e.g., it omits self-ratings when calculating average contribution) but is slightly harder to explain to students. As a result, new users can start with the normal template and move to the optimize template if they are comfortable with the more complex question type.

Reply:
@damithc Oh...I was under the impression that the optimized template is the one that I have added (i.e. the one with the distribution question) because I have set it to be the new default template like so:

![image](https://user-images.githubusercontent.com/24493463/39670547-cae9822c-5139-11e8-96e2-ebe57c3ad58e.png)

as I thought that users should by default be directed to an "optimized" version.

I see now that we have different ideas of what "optimized" means...I will update this accordingly.